### PR TITLE
PAR-2720: Add small size for text and number input

### PR DESCRIPTION
**Pull request by anisha-deverashetty**
Add small size component so we can use in FF
https://www.figma.com/file/d7rS5EkSr1xMLupCrf0VTh/FinansFront-Sketches?type=design&node-id=90-5800&mode=design&t=pPkrv2KY7Nl6FUnk-0

image

### CODE REVIEW

File: src/Nordea/Components/NumberInput.elm [hunk 1]
```diff
@@ -152,14 +165,25 @@ getStyles config =
 
             else
                 Colors.mediumGray
+
+        sizeSpecificStyling =
+            case config.size of
+                Small ->
+                    [ fontSize (rem 0.75)
+                    , height (rem 1.5)
+                    , padding2 (rem 0.25) (rem 0.5)
```
Comment (hannabergljot) [inline src/Nordea/Components/NumberInput.elm:174]: Is it correct that it has more padding than the standard size?☺️

**Review by hannabergljot — APPROVED** [review]
Really nice!!✨

File: src/Nordea/Components/NumberInput.elm (hunk 1 above)
Comment (anisha-deverashetty) [reply src/Nordea/Components/NumberInput.elm:174]: I think this is correct. Has less padding on the left and right `0.5` then the standard `0.75`. The padding on top/bottom doesn't make much difference here. I had the values from design so have included it. We can also set it `rem 0`

Comment (hannabergljot) [reply src/Nordea/Components/NumberInput.elm:174]: I see, if Lines design has 0.25 then I think we should stick with that, was just curious😁

Comment (anisha-deverashetty) [reply src/Nordea/Components/NumberInput.elm:174]: Good comment though. So, then lets keep the top/bottom padding ☺️

**Pull request merged**